cmd/documentation/documentation.go

```diff
@@ -77,15 +108,33 @@ type Attributes struct {
 }
 
 func main() {
-	verbose := flag.Bool("v", false, "Verbose output")
-
+	flag.BoolVar(&verbose, "v", false, "Verbose output")
```

Comment:
Please also keep consistency with the usage flags (Verbose output vs lowercase pass)